alchemist-physics/src/test/kotlin/it/unibo/alchemist/test/TestPhysicalPedestrians.kt

```diff
@@ -10,7 +10,7 @@ import it.unibo.alchemist.testsupport.startSimulation
 class TestPhysicalPedestrians<T, P> : StringSpec({
     "node pushes away a node in the seeking target" {
         loadYamlSimulation<T, P>("pushing_behavior.yml").startSimulation(
-            steps = 300,
+            steps = 800,
```

Comment:
why?

Reply:
When the test was written, there were no forces other than the repulsion force. I increased the number of step as it took a little longer to push the node away from its position.

Reply:
I'm just noticing the test is no longer passing anyway. I'll try to figure out why...